src/system-plugins/rov-beacon/beacon.js

```diff
@@ -0,0 +1,73 @@
+"use strict";
+const dgram = require('dgram');
+const EventEmitter = require('events');
+class Beacon extends EventEmitter{
+
+    constructor (options){
+        if(!options){
+            options = {}
+        }
+        super();
+        this.port = options.port || 8088;
+        this.broadcastAddress = options.broadcastAddress || '192.168.1.255';//;'230.185.192.108';
+        this.beaconRate = options.beaconRate || 3000;
+       
+    }
+
+    broadcast(messageFn) {
+        var self = this;
+        this.server = dgram.createSocket('udp4');
+        let server = this.server;
+        server.on('listening', () => {
+        var address = server.address();
+            server.setBroadcast(true)
+       //     server.setMulticastTTL(128);
```

Comment:
Left the hooks for Multicast as comments in case we decide to go that way in the future.